### PR TITLE
Make enzyme compatible with all React 15 Release Candidates

### PR DESCRIPTION
**Pull request by nikgraf**
Did a similar change here: https://github.com/facebook/draft-js/pull/179

### CODE REVIEW

**Comment by lelandrichardson** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -80,6 +80,6 @@
     "sinon": "^1.15.4"
   },
   "peerDependencies": {
-    "react": "0.13.x || 0.14.x || 15.* || 15.0.0-rc.2"
+    "react": "0.13.x || 0.14.x || 15.* || ^15.0.0-rc"
```
Comment (ljharb) [inline package.json]: semver ranges don't actually match against prerelease versions, so this is not a good change.

Comment (ljharb) [inline package.json]: hm, nevermind - it seems this does work. sorry about that

**Comment by nikgraf** [comment]
@lelandrichardson do you plan to publish a new release after this change? I would happy to avoid these warning …

screen shot 2016-03-20 at 13 52 53

**Comment by lelandrichardson** [comment]
@nikgraf yep!  sorry, was going to do it yesterday but got distracted. I'll do it before the end of the day :+1:

**Comment by nikgraf** [comment]
cool, thx @lelandrichardson

**Pull request merged**